versions.yaml

```diff
@@ -105,8 +105,7 @@ assets:
       snp:
         description: "VMM that uses KVM and supports AMD SEV-SNP"
         url: "https://github.com/AMDESE/qemu"
-        branch: "snp-v3"
-        commit: "ffa95097ee"
+        branch: "snp-latest" 
```

Comment:
i really think we should be pinning this to some commit that people have validated